inst/project_templates/leej22_template_files/gitignore.txt

```diff
@@ -0,0 +1,53 @@
+# ignore R data files
```

Comment:
I don't think you need a copy of this in your personal folder....you can just link to the default one

Reply:
i added one additional item to ignore : secure_data.lnk 
not sure if it's necessary but i think github logs the secure_data when the folder is first initiated

Reply:
ohhhh, interesting...I think it's logging it only when it fails (e.g. when on a network drive)